tests/testBasisLists.py

```diff
@@ -182,42 +184,42 @@ def testCentralRegularization(self):
         try:
             self.policyDF.set("centralRegularizationStencil", 1)
             ipDiffim.makeRegularizationMatrix(self.policyDF)
-        except Exception, e:  # stencil of 1 not allowed
+        except Exception as e:  # stencil of 1 not allowed
```

Comment:
Filing this under the "all these examples and tests need cleaning up" ticket I am going to make.